### DISCUSSION TITLE
Adding padding in the Action List screen

### DISCUSSION THREAD

**Discussion by MuhammadKhoshnaw** [community/community#140194] [Mobile]
Can we add some padding in the GitHub action list so the last item will be visible  currently the cancel workflow button is covering it 

![Screenshot_20240930_173209_GitHub.jpg](https://github.com/user-attachments/assets/3dcc9866-8638-4a91-8262-36b87e135e67)

**Comment by stevepopovich** [comment]
Hi @MuhammadKhoshnaw thanks for writing in. I have passed this onto the team to fix.
